packages/builder/src/components/base/FormValidationErrorList.tsx

```diff
@@ -0,0 +1,25 @@
+export default function FormValidationErrorList({ formValidation }: any) {
+  if (!formValidation.valid && formValidation.errorCount > 0)
+    return (
+      <div
```

Comment:
Nit: add `aria-live="assertive"`

Although I don't think we have cared much about this kind of things so far...